module/Finna/src/Finna/RecordDriver/SolrMarc.php

```diff
@@ -2764,14 +2764,15 @@ public function getBinding(): string
                     $this->getFieldArray($field, $subfields),
                 );
             }
-            $bindings = array_filter(array_map(
-                function ($s) {
-                    $s = mb_strtolower(mb_ereg_replace('[^A-ZÅÄÖa-zåäö]', '', $s));
-                    return $this->bindingMappings[$s] ?? null;
-                },
-                $values
-            ));
-            if (count(array_unique($bindings)) === 1) {
+            $bindings = [];
+            foreach ($values as $key) {
+                $formed = mb_strtolower(mb_ereg_replace('[^A-ZÅÄÖa-zåäö]', '', $key));
```

Comment:
Tästähän menee läpi kaikki muut kirjainmerkit, esim. kyrilliset, kreikkalaiset ja č đ ǥ jne.?

Jos tarkoitus on "poista kaikki merkit, jotka eivät ole kirjaimia":

```suggestion
                $formed = mb_strtolower(mb_ereg_replace('\P{L}', '', $key));
```

Ks. [dokumentaatio](https://www.php.net/manual/en/regexp.reference.unicode.php)

Reply:
Ne menee joka tapauksessa ton `$this->bindingMappings` -mappauksen kautta, niin mitään muuta ei voi olla.